wallet/src/de/schildbach/wallet/service/ServiceGenerator.java

```diff
@@ -0,0 +1,40 @@
+package de.schildbach.wallet.service;
+
+import java.util.concurrent.TimeUnit;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+
+public class ServiceGenerator {
+
+    private static final String TAG = "ServiceGenerator";
+    private static final String API_BASE_URL = "http://woc.reference.genitrust.com/";
```

Comment:
Great! Let's put (underneath) a commented version of this string for people to use when working with the production server.